src/main/java/io/strimzi/kafka/bridge/mqtt/utils/MappingRulesLoader.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.strimzi.kafka.bridge.mqtt.core.MqttKafkaMapper;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Helper class to Load the rules from the configuration file
+ */
+public class MappingRulesLoader {
+
+    private static final MappingRulesLoader instance = new MappingRulesLoader();
+    // path of the topic mapping rule file
+    private String mapperRuleFilePath;
+
+    /**
+     * Private constructor
+     */
+    private MappingRulesLoader() {
+    }
+
+    /**
+     * Get the singleton instance of the MappingRulesLoader
+     *
+     * @return the singleton instance of the MappingRulesLoader
+     */
+    public static MappingRulesLoader getInstance() {
+        return instance;
+    }
```

Comment:
Are you are following singleton pattern to ensure only one instance of MappingRulesLoader is created?

Is it really necessary for this case?

Reply:
I have answered [here](https://github.com/strimzi/strimzi-mqtt-bridge/pull/8#discussion_r1240604882)